spec/controllers/students_api_controller_spec.rb

```diff
@@ -9,7 +9,9 @@
         first_name: 'Agustin',
         last_name: 'Pina',
         email: 'agus@mumuki.org',
-        uid: 'agus@mumuki.org'
+        uid: 'agus@mumuki.org',
+        gender: 'male',
+        birthdate: '10/10/2010'
```

Comment:
I'd use a date where day and month are non-ambiguous

Reply:
Also, we should prefer using ISO date format.

Reply:
Done!